src/utilities/api/interactionManager.ts

```diff
@@ -500,3 +501,9 @@ export const getVoucherMetadata = async () =>
 
     .then((res) => res)
     .catch((err) => err);
+
+export const getTotalContributions = async () =>
+  axios
+    .get(totalContributions)
+    .then((res) => res)
+    .catch((err) => err);
```

Comment:
I know we do this a lot, but I'm personally a fan of not catching and doing nothing here, but rather catching in the underlying consuming component and actually doing something about it -- not sure what that would be in this case though

Reply:
Yeah agreed, it's janky that we just return an error and nothing else. We'd have to go through all endpoints we call and see what needs to be updated on the UI side to reflect the error if we do so though so idk.